schemas.py

```diff
@@ -0,0 +1,33 @@
+from datetime import date
+
+from pydantic import BaseModel
+
+
+class AuthorNoId(BaseModel):
+    name: str
+    bio: str
+
+    class Config:
+        orm_mode = True
+
+
+class AuthorWithId(AuthorNoId):
+    id: int
+
+
+class Book(BaseModel):
+    title: str
+    summary: str
+    publication_date: date
+
+    class Config:
+        orm_mode = True
```

Comment:
do you even use this schema? Usage as a schema to inherit from does not count here in this case
Also:
Generally, you should not use `orm_mode=True` like this. I mean that this option should be set on demand in places that really need it. Because with this option pydantic will use less efficient algorithm to process date upon validation and so on